.spelling

```diff
@@ -0,0 +1,111 @@
+Jetpack
+VaultPress
+WordPress
+Packagist
+WP-CLI
+rsync
+wp-admin
+config
+WordPress.com
+Jetpack-CLI
+a12s
+PHPUnit
+xdebug
+MailDev
+NodeJS
+ngrok
+http
+localhost
+multisite
+MySQL
+tunnelling
+namespace
+Automattic
+Automattician
+sftp
+PhpStorm
+wp-content
+jetpack
+wordpress
+ok
+vscode
+nvm
+phpunit
+repo
+Node.js
+i.e.
+e.g.
+keychain
+GitHub.com
+WooCommerce
+CodeSniffer
+macOS
+Homebrew
+installable
+ESLint
+rulesets
+ruleset
+PHPStorm
+hostname
+rebasing
+rebase
+unreviewed
+triaged
+jQuery
+nonces
+wp-json
+stringified
+Akismet
+GlotPress
+validator
+iframe
+Lodash
+enqueued
+polyfill
+shortlinks
+i18n
+stylesheet
+autoload
+autoloader
+webhook
+ngrok.com
+Customizer's
+error_log
+SCRIPT_DEBUG
+JETPACK_DEV_DEBUG
+jetpack_development_mode
+admin.php
+wp_localize_script
+ - docker/README.md
+php-debug
+pathMappings
+v1.3
+eu
+ap
+au
+example.ngrok.io
+www
+html
+ - docs/development-environment.md
+JETPACK__SANDBOX_DOMAIN
+README.md
+WP_DEBUG
+mycooltestsite.local
+ - docs/pull-request.md
+Pri
+ - docs/release-management.md
+Subissue
+ - docs/rest-api.md
+akismet
+jurassic.ninja
+ninjastic
+v4
+url
+options_or_modules
+ - extensions/README.md
```

Comment:
What does it mean when a full file name is added like this?

Reply:
That means it's limited to that file.